mcs/class/corlib/System.Globalization/CultureInfo.cs

```diff
@@ -93,6 +94,28 @@ struct Data {
 			public byte list_sep;
 		}
 
+		/* Native struct is culture-info.h : CultureInfoEntry */
+		const int NUM_CALENDARS = 4;
+		unsafe struct NativeCultureInfo {
```

Comment:
Please move this to separate class/struct (with icalls). That'd help me to migrate to RS CultureInfo (which is still on my todo list). The goal is to have the data part in CultureData class.